easybuild/easyconfigs/a/avrocpp/avrocpp-1.11.1-GCC-11.2.0.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'avrocpp'
+version = '1.11.1'
+
+homepage = 'https://avro.apache.org'
+description = """Avro is a data serialization system."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/apache/avro/archive/refs/tags/']
+sources = ['release-%(version)s.tar.gz']
+checksums = ['599f96bb405f72a35154b2477caa6254d723bb4e3f6a0e54e9ae540664321752']
+
+builddependencies = [
+    ('binutils', '2.37', '', ('GCCcore', '11.2.0')),
+    ('CMake', '3.22.1', '', ('GCCcore', '11.2.0')),
+]
+dependencies = [
+    ('Boost', '1.77.0'),
+]
+
+srcdir = "%(builddir)s/avro-release-%(version)s/lang/c++"
```

this path can be relative to unpacked sources:

```suggestion
srcdir = 'lang/c++'